src/main/java/de/caritas/cob/mailservice/api/service/DefaultTranslationsService.java

```diff
@@ -1,22 +1,33 @@
 package de.caritas.cob.mailservice.api.service;
 
 import de.caritas.cob.mailservice.api.model.Dialect;
-import de.caritas.cob.mailservice.config.apiclient.TranlationMangementServiceApiClient;
+import de.caritas.cob.mailservice.config.apiclient.TranslationManagementServiceApiClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.util.List;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 @Slf4j
 @Service
 public class DefaultTranslationsService {
 
+  @Value("${template.custom.default.translations.path}")
+  private String customTranslationsPath;
+
+  @Value("${template.use.custom.resources.path}")
+  private boolean useCustomResourcesPath;
```

Comment:
before PR is getting approved, please create custom image and install it on the cluster to test it works fine with the k8s changes that are already merged